SimpleImageProcessor/mainwindow.h

```diff
@@ -15,8 +17,28 @@ class MainWindow : public QMainWindow
     explicit MainWindow(QWidget *parent = 0);
     ~MainWindow();
 
+private slots:
+    void on_actionOpen_triggered();
+
+    void on_actionSave_triggered();
+
+    void on_actionSave_As_triggered();
+
+    void on_actionBlur_triggered();
+
+    void on_actionSharpen_triggered();
+
+    void on_actionTo_Greyscale_triggered();
+
+    void on_actionShow_Greylevel_Histogram_triggered();
+
+    void on_actionOtsu_s_method_triggered();
+
 private:
     Ui::MainWindow *ui;
+    QGraphicsScene *currScene;
+    QPixmap * currImg;
```

Comment:
Cheteiki dokumentaciqta se nataknah na towa - There are functions to convert between QImage and QPixmap. Typically, the QImage class is used to load an image file, optionally manipulating the image data, before the QImage object is converted into a QPixmap to be shown on screen. Alternatively, if no manipulation is desired, the image file can be loaded directly into a QPixmap. (http://doc.qt.io/qt-5/qpixmap.html#details)